.github/workflows/userbenchmark-regression-detector.yml

```diff
@@ -24,6 +24,17 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: benchmark
+      - name: Clean workspace
+        shell: bash
```

Comment:
We can move it to the end of this workflow and add `if: always()` to make sure we run it even if the previous steps fail.